packages/md-enhance/src/node/markdown-it/playground.ts

```diff
@@ -0,0 +1,146 @@
+import { hash } from "@vuepress/utils";
```

Comment:
You may rewrite this plugin by editing currect tab plugin, we may have `@file` `@setting` and other sperator instead playground container.